buildcontext/resolver.go

```diff
@@ -58,17 +58,13 @@ type Resolver struct {
 }
 
 // NewResolver returns a new NewResolver.
-func NewResolver(cleanCollection *cleanup.Collection, gitLookup *GitLookup, console conslogging.ConsoleLogger, featureFlagOverrides, gitBranchOverride, gitLFSInclude string) *Resolver {
-	return NewResolverCustomGit(cleanCollection, gitLookup, console, featureFlagOverrides, gitBranchOverride, gitLFSInclude, "")
-}
-
-// NewResolverCustomGit returns a new Resolver instance with a custom gitImage.
-func NewResolverCustomGit(cleanCollection *cleanup.Collection, gitLookup *GitLookup, console conslogging.ConsoleLogger, featureFlagOverrides, gitBranchOverride, gitLFSInclude, gitImage string) *Resolver {
```

Comment:
I ended up deleting this function since it was only used in one place.

eventually having an optsFunc type might be the better way to go, but for now we can just pass the `""` since we have some users blocked waiting for better git logs.